development/image-syncer/external-images.yaml

```
@@ -45,8 +45,8 @@ images:
 - source: "quay.io/prometheus-operator/prometheus-operator:v0.61.1"
 - source: "quay.io/prometheus/alertmanager:v0.24.0"
 - source: "quay.io/prometheus/prometheus:v2.40.6"
-- source: "nats@sha256:406dffdd6635f552a4102d89dea2ec47c566a4a3c77a55f174cf7e01163cb935"
-  tag: "2.9.8-alpine3.16"
+- source: "nats@sha256:4455732015639b8c2fa8e9020f3d17e8a209f9d59543292710768af952148382"
```

I have updated the image syncer to support multi-arch images, now the syncer expects SHA of the whole image index:
https://hub.docker.com/layers/library/nats/2.9.9-alpine3.17/images/sha256?context=explore
```suggestion
- source: "nats@sha256:c0ffa1bed32d05eebcbc9892ecfa0c37ea9f1139ef480d1bc6c8c031a8d2a78e"
```